youtube_dl/extractor/heise.py

```diff
@@ -72,11 +70,11 @@ def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
-        title = self._html_search_meta('fulltitle', webpage, default=None)
-        if not title or title == "c't":
+        title = self._og_search_title(webpage)
+        if not title:
             title = self._search_regex(
                 r'<div[^>]+class="videoplayerjw"[^>]+data-title="([^"]+)"',
-                webpage, 'title')
+                webpage, 'title', default=None)
```

Comment:
Nothing changed.

Reply:
What do you mean, "nothing changed"? If I use the version on master, the title extraction still fails for some videos (the latest c't uplink episode worked out just fine), especially the one in the unit test.

Regards

Reply:
>Adding another fallback here does not make much sense since title is not used when delegated to kaltura anyway.

Reply:
Anyway as said, If i don't do so, some videos can't be downloaded, because the title extraction fails. Also, not all videos get directed to KalturaIE, I think (can't provide an example right now though).

Is there any way of giving the extracted title to KalturaIE?

Regards!